docker/app/test_cache.py

```
@@ -0,0 +1,56 @@
+
+
+if __name__ == "__main__":
+    app.run()
+'''
+
+
+def test_cache_exports_html_and_markdown(tmp_path):
+    notebook = tmp_path / "nb.py"
+    notebook.write_text(NOTEBOOK)
+
+    result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)